site/src/contexts/useProxyLatency.ts

```diff
@@ -73,6 +74,8 @@ export const useProxyLatency = (
 		new Date(new Date().getTime() - proxyIntervalSeconds * 1000).toISOString(),
 	);
 
+	const [loaded, setLoaded] = useState(false);
```

Comment:
This is how I track when the latencies are done loading, since they all come back async